slices/perl.yaml

```diff
@@ -0,0 +1,51 @@
+package: perl
+
+essential:
+  - perl_copyright
+
+slices:
+  bins:
+    essential:
+      - libperl5.38t64_bins
+      - perl-base_bins
+      - perl-modules-5.38_modules
+      - perl_config
+    contents:
+      /usr/bin/corelist:
+      /usr/bin/cpan:
+      /usr/bin/enc2xs:
+      /usr/bin/encguess:
+      /usr/bin/h2ph:
+      /usr/bin/h2xs:
+      /usr/bin/instmodsh:
+      /usr/bin/json_pp:
+      /usr/bin/libnetcfg:
+      # /usr/bin/perlbug:
```

Comment:
Although this is a hardlink, it's a regular file in the deb. So let's include this one.